cpp/src/arrow/vendored/uriparser/UriConfig.h.in

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
We don't need this file. Because we bundle generated `config.h`.

Reply:
True. I'll remove it.